EIPS/eip-1193.md

```diff
@@ -29,7 +29,7 @@ Makes an Ethereum RPC method call.
 ```typescript
 type RequestParams = Array<any> | { [key: string]: any };
 
-ethereum.request(method: string, params?: RequestParams): Promise<unknown>;
+Provider.request(method: string, params?: RequestParams): Promise<unknown>;
```

Comment:
There's another use of `ethereum` on line 38, just below this one

Reply:
cab3d7a